src/scripts/html-transforms.js

```diff
@@ -1,11 +1,17 @@
+/**
+ * Takes two transforms returns a new function that takes an html document,
+ * runs the two transforms functions on it, and returns the the resulting document.
+ * @param {array} ...transforms The html document to transform.
```

Comment:
The right way to document this is:

```suggestion
 * @param {...(document: HTMLDocument) => HTMLDocument} transforms The HTML transforms to combine.
```

- For a variable number of arguments, use `{...TypeName}`.
- Since the type is a function, `TypeName` is `(document: HTMLDocument) => HTMLDocument`. That’s saying:
    - It’s a function: `(something) => somethingElse`
    - It takes an `HTMLDocument` instance: `(document: HTMLDocument) => somethingElse`
    - It returns an `HTMLDocument` instance: `(document: HTMLDocument) => HTMLDocument`
- And the actual name of the argument is just `transforms` (no ellipsis).